vumi/persist/tests/test_model.py

```diff
@@ -850,6 +850,21 @@ def test_vumimessage_field(self):
         m1.msg = msg2
         self.assertTrue("extra" not in m1.msg)
 
+    @Manager.calls_manager
+    def test_vumimessage_field_excludes_cache(self):
+        msg_model = self.manager.proxy(VumiMessageModel)
+        cache_attr = TransportUserMessage._CACHE_ATTRIBUTE
+        msg = self.mkmsg(extra="bar", __cache__={"cache": "me"})
```

Comment:
Maybe add the cache data using `msg.cache` after creation instead of specifying `__cache__=` in the `mkmsg()` call? That would avoid having to use knowledge of the dict key here.